agent/agentbootstrap/bootstrap.go

```diff
@@ -108,11 +111,23 @@ func InitializeState(
 	info.Tag = nil
 	info.Password = c.OldPassword()
 
+	// Add the controller model cloud and credential to the database.
+	cloudCred, cloudCredTag, err := getCloudCredential(adminUser, args)
+	if err != nil {
+		return nil, errors.Trace(err)
```

Comment:
`return nil, fmt.Errorf("getting cloud credentials from args: %w", err)`

This doesn't help me at the moment.

Reply:
The called function emits a useful error message. Using fmt.Errorf loses the call stack. I'll annotate but we do need to avoid stuttering which is common problem in many juju errors